### PR TITLE
add globalization

### PR DESCRIPTION
**Pull request by loay**
Connect to strongloop/loopback/issues/2422

### CODE REVIEW

**Comment by loay** [comment]
@superkhau @0candy 
PTAL

File: test-server/index.js [hunk 1]
```diff
@@ -105,5 +108,5 @@ app.use(require('morgan')('loopback> :method :url :status'));
 app.enableAuth();
 app.use(loopback.rest());
 app.listen(3000, function() {
-  console.log('LoopBack test server listening on http://localhost:3000/');
+  console.log(g.f('LoopBack test server listening on {{http://localhost:3000/}}'));
```
Comment (0candy) [inline test-server/index.js]: Should add {{LoopBack}} for LoopBack

File: test-server/remoting/index.js [hunk 2]
```diff
@@ -22,7 +24,6 @@ app.use(remotes.handler('rest'));
 var server = require('http')
   .createServer(app)
   .listen(3001, function() {
-    console.log(
-      'strong-remoting test server listening on http://localhost:3001/');
+    console.log(g.f(
+      'strong-remoting test server listening on {{http://localhost:3001/}}'));
```
Comment (0candy) [inline test-server/remoting/index.js]: Should add {{strong-remoting}} for strong-remoting

**Comment by 0candy** [comment]
You have to run slt-globalize -e again for intl/en/messages.json to get updated with the brackets change.

**Comment by loay** [comment]
@0candy Fixed

**Comment by 0candy** [comment]
LGTM

**Pull request merged**